src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -1,15 +1,12 @@
-import React, {useContext, useEffect, useRef, useState} from "react";
-
-import Editor, {loader} from "@monaco-editor/react";
+import React, { useContext, useEffect, useRef, useState } from "react";
 import * as monaco from "monaco-editor";
 import PropTypes from "prop-types";
 
-import {THEME_STATES} from "../../../ThemeContext/THEME_STATES";
-import {ThemeContext} from "../../../ThemeContext/ThemeContext";
+import { THEME_STATES } from "../../../ThemeContext/THEME_STATES";
```

Comment:
Let's apply the eslint config file so that when we use a code formatter, we can avoid changing lines that are unrelated to the PR.